src/features/hotspots/details/HotspotDetailChart.tsx

```diff
@@ -115,12 +116,22 @@ const HotspotDetailChart = ({
   const { black, grayLight, grayMain, purpleMain, greenOnline } = useColors()
   const [focusedData, setFocusedData] = useState<ChartData | null>(null)
 
-  const onFocus = useCallback((chartData: ChartData | null) => {
+  const onFocus = useCallback(async (chartData: ChartData | null) => {
     if (Platform.OS === 'ios') {
       // this animation causes layout issues on Android
       animateTransition()
     }
-    setFocusedData(chartData)
+
+    if (!chartData) {
+      setFocusedData(null)
+      return
+    }
+
+    const label = await DateModule.formatDate(
+      chartData.label,
+      chartData.showTime ? 'MMM d h:mma' : 'EEE MMM d',
```

Comment:
One last thing here, for Germany again on iOS usually the time would convert to 24 hour time instead of AM/PM.

I don't see that happening? If it isn't possible, I think its good enough though.

Reply:
I think it's that same issue of the locale being set to `en_DE` vs `de_DE`

Reply:
which we don't have control over

Reply:
it's a system setting

Reply:
Ah ok, no problem then